junit-vintage-engine/junit-vintage-engine.gradle

```diff
@@ -18,7 +18,8 @@ dependencies {
 	testImplementation(project(':junit-jupiter-api'))
 	testImplementation(project(':junit-platform-runner'))
 	testImplementation(project(path: ':junit-jupiter-engine', configuration: 'testArtifacts'))
-	testImplementation(project(path: ':junit-platform-engine', configuration: 'testArtifacts'))
+	testImplementation(project(':junit-platform-testkit'))
+	testImplementation(project(path: ':junit-platform-testkit', configuration: 'testArtifacts'))
```

Comment:
Typo? The original line `testImplementation(project(path: ':junit-platform-engine', configuration: 'testArtifacts'))` should be not replaced, afaikt.

Reply:
Yep, sorry that was a typo, will correct!

Reply:
Er, wait sorry I thought this was a typo too but I realized that I actually moved part of the original test classes from `junit-platform-engine` into the `testkit` module, so this is correct now since `junit-platform-engine` doesn't have any test artifacts anymore (since they were replaced with what's in the `testkit`

Reply:
Mh. Vintage Engine does not compile: https://travis-ci.org/junit-team/junit5/jobs/393787737#L1591
Why?

Reply:
That's really strange, I can't repro it on my machine... looking into it further now.